src/caret_analyze/plot/timeseries/timeseries_plot_factory.py

```diff
@@ -54,7 +54,8 @@ def create_instance(
         visualize_lib : VisualizeLibInterface
             Instance of VisualizeLibInterface used for visualization.
         case : str
-            Parameter specifying best, worst or all. Use to create Response time timeseries graph.
+            Parameter specifying best, worst, worst_in_input, or all.
```

Comment:
```suggestion
            Parameter specifying best, worst, worst-in-input, or all.
```

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.

Reply:
Not fixed.